modules/virt-hot-plugging-a-virtual-disk-using-the-cli.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk-using-the-cli_{context}"]
+= Hot-plugging a virtual disk using the CLI
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claim (PVC) available for hot-plugging.
+
+Hot-plug a virtual disk by running the following command:
```

Comment:
This should be the first step of the procedure instead of an intro sentence. Same comment for the unplugging procedure.

Reply:
Done